test/test.config.js

```diff
@@ -36,3 +36,6 @@ roles['bedrock-test.regular'] = {
     permissions.EDV_CONFIG_REMOVE.id
   ]
 };
+
+config.profile.kms.baseUrl = `${config.server.baseUri}/kms`;
+config.profile.kms.ipAllowList = ['127.0.0.1/32'];
```

Comment:
This is fine for now, but the plan is to move this into keystore configs, right?

Reply:
this informs what ipAllowList gets into the keystore configs.  

See: https://github.com/digitalbazaar/bedrock-profile/pull/67

specifically: https://github.com/digitalbazaar/bedrock-profile/compare/refactor-kms?expand=1#diff-7f2cfbda4d88af2c71bb531dea727cc6cf1b9c6acdf67b2b96083075d7148f38R20-R39

Reply:
Aha! Thank you.